app/controllers/api_controller/parser/request_adapter.rb

```diff
@@ -35,7 +47,7 @@ def collection
       end
 
       def c_suffix
-        @params[:c_suffix] || c_path_parts[1..-1].join('/')
+        @params[:c_suffix] || Array(c_path_parts[1..-1]).join('/')
```

Comment:
I'm not sure why this is needed...that is when can a slice of an Array ever *not* be an Array?